test/configCases/module-name/different-issuers-for-same-module/test.js

```diff
@@ -0,0 +1,8 @@
+it("should assign different names to the same module with different issuers ", function() {
+	var fs = require("fs");
+	var path = require("path");
+	var bundle = fs.readFileSync(path.join(__dirname, "main.js"), "utf-8");
+	expect(bundle.match(/"\.\/c\.js\?\w{4}":/g)).toHaveLength(2);
```

Comment:
These 4 lines can be removed. And instead test the exported value of a and b to match the expected id.